### DISCUSSION TITLE
Move github version comparison URLs into version header?

### DISCUSSION THREAD

**Discussion by TwitchBronBron** [olivierlacan/keep-a-changelog#534] [Ideas]
I maintain 10+ open source projects and I regularly forget to add the github version comparison URL at the bottom of the file. In my projects I moved the URL inline with the version, and now it's almost impossible to forget because it's right in front of me. I'm happy to submit a PR with these changes if you agree with the concept.

# Proposal:
Update the spec to include git diff URL inline with version header.

**change this:**
```markdown
## [0.1.0] - 2015-10-06
...many lines later
[0.1.0]: https://github.com/olivierlacan/keep-a-changelog/compare/v0.0.8...v0.1.0
```

**to this:**
```markdown
## [0.1.0](https://github.com/olivierlacan/keep-a-changelog/compare/v0.0.8...v0.1.0) - 2015-10-06
```

**Comment by pcolby** [comment]
It's probably a just personal preference, but I think its better the way it is now.  My editor shows me if the link is recognised (as a link) or not, so I tend not to have that issue.  And I feel (personal opinion) like having the deep links at the end better adheres to Keep a Changlog's [Guiding Principle #1](https://keepachangelog.com/en/1.0.0/#principles):

> Changelogs are for humans, not machines.

As Changelogs are written a couple of times (per release), but read many times, I think the current apporoach is more human-friendly? 🙂 Could just be me of course.
